batcher/aligned/src/main.rs

```diff
@@ -528,6 +596,51 @@ async fn main() -> Result<(), AlignedError> {
                 }
             }
         }
+        GetUserNonceFromEthereum(args) => {
+            let address = H160::from_str(&args.address).unwrap();
+            let network = args.network.into();
+            match get_nonce_from_ethereum(&args.eth_rpc_url, address, network).await {
+                Ok(nonce) => {
+                    info!(
+                        "Nonce for address {} in BatcherPaymentService contract is {}",
+                        address, nonce
+                    );
+                }
+                Err(e) => {
+                    error!("Error while getting nonce: {:?}", e);
+                    return Ok(());
+                }
+            }
+        }
+        GetAmountOfProofsInBatcherQueue(args) => {
+            let address = H160::from_str(&args.address).unwrap();
+            let network = args.network.into();
+            let ethereum_nonce =
+                match get_nonce_from_ethereum(&args.eth_rpc_url, address, network).await {
+                    Ok(nonce) => nonce,
+                    Err(e) => {
+                        error!("Error while getting nonce: {:?}", e);
+                        return Ok(());
+                    }
+                };
+            let batcher_nonce = match get_nonce_from_batcher(&args.batcher_url, address).await {
+                Ok(nonce) => nonce,
+                Err(e) => {
+                    error!("Error while getting nonce: {:?}", e);
+                    return Ok(());
+                }
+            };
```

Comment:
These can probably be done simultaneously. Maybe an issue to consider it might be good. Non-blocking BTW.

Reply:
```suggestion
            let (ethereum_nonce, batcher_nonce) = match future::try_join(
                get_nonce_from_ethereum(&args.eth_rpc_url, address, network),
                get_nonce_from_batcher(&args.batcher_url, address),
            ).await {
                Ok(nonce) => nonce,
                Err(e) => {
                    error!("Error while getting nonce: {:?}", e);
                    return Ok(());
                }
            };
```

Reply:
Took this a step further

```
Tried taking this a step further:
            let Ok((ethereum_nonce, batcher_nonce)) = future::try_join(
                get_nonce_from_ethereum(&args.eth_rpc_url, address, network),
                get_nonce_from_batcher(&args.batcher_url, address),
            )
            .await
            .map_err(|e| error!("Error while getting nonce: {:?}", e)) else {
                return Ok(());
            };
```